### ISSUE TITLE
Fixing daisy lock icon (incorrect img path)

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#4807]
I think we may have an incorrect image path which you can see in the editions table beneath the borrow button:
`/images/icons/icon-encrypto-sm.png` should be `/static/images/icons/icon-encrypto-sm.png`

### Evidence / Screenshot (if possible)

see:
http://staging.openlibrary.org/books/OL9240710M/Understanding_Comics

![staging openlibrary org_books_OL9240710M_Understanding_Comics](https://user-images.githubusercontent.com/978325/111196540-a2df9c80-857a-11eb-8f36-9bf510d98fdf.png)

### Relevant url?

http://staging.openlibrary.org/books/OL9240710M/Understanding_Comics

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual: 
* Expected: 

### Details

- **Logged in (Y/N)?** 
- **Browser type/version?** 
- **Operating system?** 
- **Environment (prod/dev/local)?** prod
